tests/utils/test_io.py

```diff
@@ -1,10 +1,38 @@
-import sys
 import io
+import sys
```

Comment:
Why the reorder??

Reply:
iv started running the atuoamtic import cleanup/reorder in vscode on any files i touch to get consistency.